docs/src/main/paradox/amqp.md

```diff
@@ -2,12 +2,7 @@
 
 The AMQP connector provides Akka Stream sources and sinks to connect to AMQP servers.
 
-### Reported issues
-
-[Tagged issues at Github](https://github.com/akka/alpakka/labels/p%3Aamqp)
```

Comment:
Please keep the reported issues section.

Reply:
Done!